### PR TITLE
managed documentation 

### LINKED ISSUE #21: Documentation to give an overview of the managed.hopsworks.ai dashboard.

### LINKED ISSUE #22: Documentations for create and restore backups

### LINKED ISSUE #23: Documentation to give an overview of the settings page.

### PR DESCRIPTION
**Pull request by berthoug**
Fixes #21 
Fixes #22 
Fixes #23

### CODE REVIEW

File: docs/setup_installation/common/backup.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+      <img style="border: 1px solid #000" src="../../../assets/images/setup_installation/managed/common/backup/ongoing_backup.png" alt="Backup ongoing">
+    </a>
+    <figcaption>Backup ongoing</figcaption>
+  </figure>
+</p>
+
+Once the backup is taken your cluster will be back up and running and ready to use.
+
+## Restoring a backup
+Go to the Backup tab of the dashboard (left menu (1)) to the list of all the backups. This list is organized by clusters. For each of the clusters, you can see the state of the cluster (2) and the list of backups for this cluster (3).
```
Comment (steffengr) [inline docs/setup_installation/common/backup.md:68]: organized by cluster without s if I'm not mistaken

File: docs/setup_installation/common/dashboard.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+  <figure>
+    <a  href="../../../assets/images/setup_installation/managed/common/dashboard/general_tab.png">
+      <img style="border: 1px solid #000" src="../../../assets/images/setup_installation/managed/common/dashboard/general_tab.png" alt="General tab">
+    </a>
+    <figcaption>General tab</figcaption>
+  </figure>
+</p>
+
+### Manage the services in the services tab
+The services tab shows which services ports are open to the internet on your cluster. More details can be found in the [services documentation](./services.md).
```
Comment (steffengr) [inline docs/setup_installation/common/dashboard.md:91]: Maybe service ports instead of plural?

File: docs/setup_installation/common/dashboard.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  <figure>
+    <a  href="../../../assets/images/setup_installation/managed/common/dashboard/services_tab.png">
+      <img style="border: 1px solid #000" src="../../../assets/images/setup_installation/managed/common/dashboard/services_tab.png" alt="Services tab">
+    </a>
+    <figcaption>Services tab</figcaption>
+  </figure>
+</p>
+
+### Get information about your cluster status in the Console tab
+The console tab display more detailed information about the current state of your cluster. If your cluster is running and everything is as planned it will only say "everything is ok". But, if something failed, this is where you will find more details about the error.
```
Comment (steffengr) [inline docs/setup_installation/common/dashboard.md:103]: We say status in the headline but state in the text

File: docs/setup_installation/common/settings.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/setup_installation/managed/common/settings/settings.png">
+      <img style="border: 1px solid #000" src="../../../assets/images/setup_installation/managed/common/settings/settings.png" alt="Getting to the settings page">
+    </a>
+    <figcaption>Getting to the settings page</figcaption>
+  </figure>
+</p>
+
+## Manage the connection to your cloud accounts in Cloud Accounts
```
Comment (steffengr) [inline docs/setup_installation/common/settings.md:20]: Not sure if in is correct here or if it should be under Cloud Accounts

Comment (steffengr) [reply docs/setup_installation/common/settings.md:20]: Same with some following headlines

**Review by steffengr — CHANGES_REQUESTED** [review]

File: docs/setup_installation/common/backup.md [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+      <img style="border: 1px solid #000" src="../../../assets/images/setup_installation/managed/common/backup/backup_delete_confirm.png" alt="Confirm backup deletion">
+    </a>
+    <figcaption>Confirm backup deletion</figcaption>
+  </figure>
+</p>
+
+The backup will then be deleted.
+
+## Conclusion
+During this tutorial, you have created a backup, restored a cluster from this backup, checked the information about this backup, and finaly deleted the backup.
```
[the remark this answers is not in the document]
Comment (berthoug) [reply docs/setup_installation/common/backup.md:169]: I don't get how gramarly missed this one.

**Review by steffengr — APPROVED** [review]

**Pull request merged**